cost-analyzer/templates/_helpers.tpl

```diff
@@ -1256,6 +1270,14 @@ Begin Kubecost 2.0 templates
     {{- end}}
     {{- end }}
     {{- end }}
+    {{- if (.Values.enterpriseCustomPricing).enabled }}
+    - name: ENTERPRISE_CUSTOM_PRICING_ENABLED
+      value: "true"
+    - name: ENTERPRISE_CUSTOM_PRICING_CSV_LOCATION_URI
+      value: {{ .Values.enterpriseCustomPricing.location.URI }}
+    - name: ENTERPRISE_CUSTOM_PRICING_APPLY_RETROACTIVELY
+      value: {{ (quote .Values.enterpriseCustomPricing.applyRetroactively) | default (quote false) }}
```

Comment:
Minor comment. My preference is to always set defaults in the `values.yaml` as opposed to in the template here. That way it's more transparent to the user what the default behavior should be.

Reply:
Makes sense. I'm happy to follow whatever the best practices are. I just need to be re-trained after years of helm slop!